sdkclient/tx/client.go

```diff
@@ -110,5 +120,46 @@ func (c *Client) initTxFactory() {
 }
 
 func (c *Client) BroadcastTx(msgs ...sdk.Msg) (*sdk.TxResponse, error) {
+	c.ClientContext.From = c.keyringRecord[0].Name
+	c.ClientContext.FromName = c.keyringRecord[0].Name
+	c.ClientContext.FromAddress, _ = c.keyringRecord[0].GetAddress()
 	return BroadcastTx(*c.ClientContext, *c.txFactory, msgs...)
 }
+
+func (c *Client) BroadcastTxVotes(proposalID uint64) error {
+	for index := range c.keyringRecord {
+		voter, err := c.keyringRecord[index].GetAddress()
+		if err != nil {
+			return err
+		}
+
+		voteType, err := govtypes.VoteOptionFromString("VOTE_OPTION_YES")
+		if err != nil {
+			return err
+		}
+
+		msg := govtypes.NewMsgVote(
+			voter,
+			proposalID,
+			voteType,
+		)
+
+		c.ClientContext.From = c.keyringRecord[index].Name
+		c.ClientContext.FromName = c.keyringRecord[index].Name
+		c.ClientContext.FromAddress, _ = c.keyringRecord[index].GetAddress()
+
+		for retry := 0; retry < 5; retry++ {
+			// retry if txs fails, because sometimes account sequence mismatch occurs due to txs pending
+			if _, err = BroadcastTx(*c.ClientContext, *c.txFactory, []sdk.Msg{msg}...); err == nil {
+				break
+			}
+			time.Sleep(time.Second * 1)
+		}
```

Comment:
let's check if err is sequence missmatch. If yes, then we continue the loop, otherwise, we break.

Reply:
Sometimes we are getting other than mismatch also (ex: timeout for tx include in the block).
We will keep this loop like this, so if we got any errors we wait and retry